scripts/release/build/build_scenario.py

```diff
@@ -1,69 +1,9 @@
 from enum import StrEnum
 
-from git import Repo
-
-from lib.base_logger import logger
-from scripts.release.constants import triggered_by_git_tag, is_evg_patch, is_running_in_evg, get_version_id, \
-    get_github_commit
-from scripts.release.version import calculate_next_version
-
-COMMIT_SHA_LENGTH = 8
-
 
 class BuildScenario(StrEnum):
     RELEASE = "release"  # Official release triggered by a git tag
     MANUAL_RELEASE = "manual_release"  # Manual release, not part of operator release cycle
     PATCH = "patch"  # CI build for a patch/pull request
     STAGING = "staging"  # CI build from a merge to the master
     DEVELOPMENT = "development"  # Local build on a developer machine
-
-    @classmethod
-    def infer_scenario_from_environment(cls) -> "BuildScenario":
```

Comment:
Have we moved this somewhere ? Or do we drop scenario inference ?

Reply:
After reading further, it was moved to `scripts/dev/contexts/evg-private-context`, is that correct ?
Is the development scenario chosen by default locally ?

Reply:
`BuildScenario` inference from env vars was a code smell, because the env vars were not visible in the script inputs. Right now you have to specify the `BuildScenario` explicitly whenever you want to build the image locally. You can use `development` of course.

If you rather want evergreen to build the images for you, then the `BuildScenario` is calculated based on evergreen env variables in `scripts/dev/contexts/evg-private-context` as you noticed.

Reply:
I'm okay to resolve. Maybe we should also track these follow ups in a Jira ticket ?

Reply:
For bugs and help tickets it makes sense, but for projects it's additional burden to duplicate what we have in the doc. I wouldn't go that way, it's unnecessary bureaucracy